integration/docker-compose.spdk.yml

```diff
@@ -16,19 +16,20 @@ services:
       opi:
         ipv4_address: 10.127.127.4
     command: |
-      sh -c 'sync; echo 1 | sudo tee /proc/sys/vm/drop_caches  && \
+      sh -x -c 'sync; echo 1 > /proc/sys/vm/drop_caches  && \
             echo 1024 > /proc/sys/vm/nr_hugepages && \
             grep "" /sys/kernel/mm/hugepages/hugepages-*/nr_hugepages && \
             /root/spdk/build/bin/spdk_tgt -m 0x1 -s 512 2>&1 & \
             echo wait 5s... && sleep 5s && cd /root/spdk/scripts && \
+            for i in `seq 1 10`; do sleep 1; ./rpc.py spdk_get_version || continue; done  && \
             ./rpc.py bdev_malloc_create -b Malloc0 64 512 && \
             ./rpc.py bdev_malloc_create -b Malloc1 64 512 && \
             ./rpc.py nvmf_create_transport -t TCP -u 8192 -m 4 -c 0  && \
             ./rpc.py nvmf_create_subsystem nqn.2016-06.io.spdk:cnode1 -a -s SPDK00000000000001 -d SPDK_Controller1  && \
-            ./rpc.py nvmf_subsystem_add_listener nqn.2016-06.io.spdk:cnode1 -t tcp -a 0.0.0.0 -s 4420  && \
+            ./rpc.py nvmf_subsystem_add_listener nqn.2016-06.io.spdk:cnode1 -t tcp -a 10.127.127.4 -s 4420  && \
```

Comment:
Should we use hostname here in case IP address changes? Maybe something for another PR so we can unblock by merging this one now.

Reply:
I tried here and no luck, will give it another try with a separate PR